docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+services:
+  app:
+    depends_on:
+      - mongodb
+    build: .
+    restart: unless-stopped
+    environment:
+      # You can leave this as-is.
```

Comment:
Could you change from DB to MONGO? Also add the ability to use MONGO_URL instead of seperating them?